### ISSUE TITLE
Should be `loss * self.eps/(c-1)`? 

### ISSUE THREAD

**Issue by CiaoHe** [ZhugeKongan/torch-template-for-deep-learning#2]
Hi,

Based on my understanding, the label smooth loss should be constructed like: for the non-target label, the corresponding value should be set as `\epsilon / (num_classes-1)` 

https://github.com/ZhugeKongan/torch-template-for-deep-learning/blob/4a84444066456c7bb27daeec5e85d512eccc88cf/autoaug/label_smoothing.py#L19

So, this line might be correct as  `loss*self.eps/(c-1)`?

**Comment by ZhugeKongan** [comment]
Yes, thank you for your feedback. There are still many details that need to be continuously improved.
